src/components/App.jsx

```diff
@@ -16,6 +17,7 @@ import ProjectHeader from './ProjectHeader';
 class App extends React.Component {
   componentWillMount() {
     this.props.dispatch(fetchProject());
+    this.props.dispatch(fetchWorkflow());
```

Comment:
My primary comment is that we need to keep an eye on this. Previously, we've hardcoded the workflow that our projects use within our config.js file. However, moving forward, the 'standard' Zooniverse practice is to set the Workflow ID within the Project settings, so we'll need to do something like... `fetchProject().then((proj)=>{ fetchWorkflow(proj.workflowid) })`

There are reasons why we _would want_ to use hardcoded Workflow IDs - e.g. with ASM, users in different Splits get different workflows - but they don't seem applicable to Scribes.

For now.

As far as I know.

Dun dun _dun._

But anyway, important thing is that we need to keep an eye on this. I'm aware that PR #18 makes a lot of changes to the API calls, so it's best that this PR isn't changed, and that any updates done in future PRs. (If 18 doesn't take care of this already.)